src/AccessibilityInsights.SharedUx/Controls/SettingsTabs/ConnectionControl.xaml.cs

```diff
@@ -53,444 +41,96 @@ private enum States
         public Action<Action> HandleLogoutRequest { get; set; }
         public Action<bool> ShowSaveButton { get; set; }
 
-
-        /// <summary>
-        /// Avatar view model
-        /// </summary>
-        public ByteArrayViewModel vmAvatar { get; private set; } = new ByteArrayViewModel();
-
-        /// <summary>
-        /// Current user display name
-        /// </summary>
-        public static string DisplayName => BugReporter.DisplayName;
-
-        /// <summary>
-        /// Current user email
-        /// </summary>
-        public static string Email => BugReporter.Email;
-
-        /// <summary>
-        /// List of team projects
-        /// </summary>
-        public BindingList<TeamProjectViewModel> projects { get; private set; } = new BindingList<TeamProjectViewModel>();
+        IssueConfigurationControl issueConfigurationControl = null;
+        IIssueReporting selectedIssueReporter = null;
 
         #region configuration updating code
         /// <summary>
-        /// Updates the MRU list of servers from the configuration
-        /// Depending on whether we are logged in, we update the server combo box selection to
-        /// the MRU server or to the currently connected server
+        /// Initializes the view.
         /// </summary>
         /// <param name="configuration"></param>
-        public void UpdateFromConfig(ConfigurationModel configuration)
+        public void UpdateFromConfig()
```

Comment:
do we still need this method?

Reply:
Nope. Updated to directly call Initialize view.